ncl/ncl_entries/day_of_week.ipynb

```diff
@@ -0,0 +1,231 @@
+{
```

Comment:
<p>I wonder if there's a way to change the strftime representation instead of doing the days list </p><p> </p><p>(I might be thinking too hard about this)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Thank you for pointing this out! I was able to make this change without too much trouble and it is a much preferable solution in my opinion.